apps/balancer-tools/src/app/apr/(utils)/sortChartData.ts

```diff
@@ -0,0 +1,34 @@
+interface iData {
```

Comment:
not a very descriptive name, would use `AxisData`. Also, not the standard we're using for interfaces (interface names should be TitleCase and we don't prefix with `I`) see the rest of the code.

Reply:
also consider whether you really need an interface here -- couldn't you just plug x and y as two function params? Why complicate things?